packages/theme/src/theme/_forms.switch.scss

```diff
@@ -95,4 +93,8 @@ $switch-track-shadow-disabled: inset 2px 2px 7px 0 rgba(0, 0, 0, 0.2);
 			}
 		}
 	}
+	.switch-label {
+		margin-left: $padding-normal;
```

Comment:
Adding a class to the label would be like a breaking change because some apps use the theme without using components. 
And if you look at the [theme demo](http://2498.talend.surge.sh/theme/#forms), it's broken.
I know it's not sexy af to use `+ *` but it's the best way to achieve it quickly without any ponderation impact. 🤷‍♂